gatsby-node.js

```diff
@@ -401,21 +411,64 @@ const getLearnPageFilters = async () => {
     return filters;
 };
 
-exports.onCreatePage = async ({ page, actions }) => {
-    if (page.path === '/learn/') {
-        const { createPage, deletePage } = actions;
-        const allArticles = await getLearnPageArticles();
-        const featuredArticles = getFeaturedLearnArticles(allArticles);
-        const filters = await getLearnPageFilters(allArticles);
-        deletePage(page);
-        createPage({
-            ...page,
-            context: {
-                ...page.context,
-                allArticles,
-                featuredArticles,
-                filters,
-            },
-        });
+const getFeaturedArticlesForPage = async (
+    page,
+    defaultFeaturedArticles,
+    allArticles = null
+) => {
+    if (!allArticles) {
+        allArticles = await getLearnPageArticles();
+    }
+    // TODO: Verify field names with docsp
+    const featuredArticleSlugs =
+        getNestedValue(['featuredArticles', `${page}Page`], metadata) ||
+        defaultFeaturedArticles;
```

Comment:
We can merge this change while the docs platform team is determining the values they want to support for this. If we don't get anything in metadata, we will use one of the `DEFAULT` slug arrays up top (as we are now for the learn page)